main.py

```diff
@@ -211,7 +220,9 @@ def buy():
                             logger.info(f"Clearing order with a status of {order_status}.  Waiting for 'closed' status")
                             order.clear()  # reset for next iteration
                 else:
-                    logger.warning(f'{announcement_coin=} is not supported on gate io')
+                    message = f'{announcement_coin=} is not supported on gate io'
+                    logger.warning(message)
+                    send_telegram(message)
```

Comment:
I don't think that all those messages are really necessary. My assumption would be that only two cases are really necessary:
- new announcement with it's results
- heartbeat

If someone wants details about every single thing that happened, they might probably want to go into the logs themselves. You can't really do anything with an error message in Telegram anyway. Just my opinion though. :)

Reply:
I personally would like to get this information. Maybe it makes sense to have this configurable?

Reply:
You could have something like a TELEGRAM_LOG_LEVEL and then send everything to telegram that fits in that log level?
I mean, whatever idea you can think of is fair. :) The important thing is that those who don't want it need to be able to turn it off. So making all that configurable sounds like a great idea to me. :)

Reply:
Okay, I think i have a simple implementation that gives good configurability to the user https://github.com/CyberPunkMetalHead/gateio-crypto-trading-bot-binance-announcements-new-coins/pull/18/commits/1100695bf41dd934b16f97e1258ee9985d039a01

Reply:
I think your new solution is definitely better. 👍